app/design/adminhtml/default/default/template/ebizmarts/mailchimp/system/config/form/field/array_dropdown.phtml

```diff
@@ -45,16 +45,16 @@ $colspan = $colspan > 1 ? 'colspan="' . $colspan . '"' : '';
                     </span>
                 </button>
             </td>
-            <td <?php echo $this->quoteEscape($colspan) ?>>
+            <td <?php echo $this->gethelper()->mcEscapeQuote($colspan) ?>>
                 <button style=""
                         onclick=""
                         class="scalable add"
                         type="button"
-                        id="addToEndBtn<?php echo $this->quoteEscape($htmlId) ?>">
+                        id="addToEndBtn<?php echo $this->gethelper()->mcEscapeQuote($htmlId) ?>">
                     <span>
                         <span>
                             <span>
-                                <?php echo $this->quoteEscape(Mage::helper('mailchimp')->__('Add new row')) ?>
+                                <?php echo $this->gethelper()->mcEscapeQuote(Mage::helper('mailchimp')->__('Add new row')) ?>
```

Comment:
This can be more straight forward by calling only one function that uses the helper internally.